README.md

```diff
@@ -114,6 +136,14 @@ We've written a [porting guide](https://medium.com/@comma_ai/openpilot-port-guid
 
 Sadly, BMW, Audi, Volvo, and Mercedes all use [FlexRay](https://en.wikipedia.org/wiki/FlexRay) and are unlikely to be supported any time soon. We also put time into a Ford port, but the steering has a 10 second cutout limitation that makes it unusable.
 
+### What is Comma Pedal?
```

Comment:
this is a good paragraph, which is worth to keep. I'll copy paste in our internal branch. Thanks!

Reply:
So does this mean you want the rest of the readme changes removed or?